akka-http-core/src/main/resources/reference.conf

```diff
@@ -431,6 +443,18 @@ akka.http {
 
       # Enable verbose debug logging for all ingoing and outgoing frames
       log-frames = false
+
+      # When there is no data transmitted, send a HTTP/2 ping frame with this interval to make sure the connection is
+      # kept alive, 0s disables sending ping frames. Must be full seconds.
+      keepalive-time = 0s
```

Comment:
Maybe just `ping-interval`?

Reply:
The PR name is simple taken from the issue name, and is not super important.

The comment describes pretty much exactly what it does, when there is no data transmitted over the connection, a ping is sent with this interval, including first interval between last data and first ping. As soon as there is data the pinging stops, because there already is data moving across the wire so no keepalive is needed. I think interval is a better name than time.